srml/timestamp/readme.md

```diff
@@ -0,0 +1,81 @@
+# Timestamp Module
```

Comment:
i think for consistency with the rest of the codebase that the actual readme file should be named to README.adoc (AsciiDoc format) instead of being in Markdown format. The name of the file should be `README` not lowercase `readme`